channeld/channel_wire.csv

```diff
@@ -159,6 +159,8 @@ msgtype,channel_got_revoke_reply,1122
 
 # Tell peer to shut down channel.
 msgtype,channel_send_shutdown,1023
+msgdata,channel_send_shutdown,shutdown_len,u16,
```

Comment:
This parameter is not reflected in the mocks nor in your usage of `fromwire` and `towire` ?

Reply:
Yes. For u8 array, `towire` will get this parameter by `tal_count`, and `fromwire` use it to initial a u8 array by `tal_array`. The bottom details are warpped in `wire/towire.c` and `wire/fromwire.c`.